lib/bolt/task.rb

```diff
@@ -33,21 +33,26 @@ def supports_noop
     end
 
     def file_map
-      @file_map ||= files.each_with_object({}) { |file, hsh| hsh[file['name']] = file['path'] }
+      @file_map ||= files.each_with_object({}) { |file, hsh| hsh[file['name']] = file }
```

Comment:
Why change this?

Reply:
it gives us a place to override in the subclass. This is how I was lazily syncing code down.